mkdocs.yml

```
@@ -0,0 +1,6 @@
+site_name: sops-compliance-checker
+site_description: |
+    Tool that looks for SOPS files in a repository
+    and makes sure that they are configured in the desired fashion
+nav:
+  - Home: index.md
```

We don't have `index.md` yet, let's just only link the design document for now.

```suggestion
  - Design: design.md
```

---

Was thinking maybe we add index.md in the next patches. For now added design only.
I skipped schema.json as the 'fixed' version is less readable for me than the current one. We can change that later if we decide to.